src/main/java/io/github/syst3ms/skriptparser/lang/VariableString.java

```diff
@@ -99,11 +104,45 @@ private VariableString(Object[] data) {
                 }
                 data.add(expression.get());
                 i += content.get().length() + 1;
+            } else if (c == '<') {
+                if (i == charArray.length - 1) {
+                    return Optional.empty();
+                }
+                var content = StringUtils.getBracketContent(s, i + 1, '>');
+                if (content.isEmpty()) {
+                    sb.append(c);
```

Comment:
I don't know if this is a good idea. It could make some bugs very hard to detect.

Reply:
What do you mean exactly?

Reply:
Just ignoring an unmatched bracket, idk. Though, I suppose it should be fine in this case since there's no issue with nesting.